diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -109,6 +109,7 @@ enum class Warnings(private val canBeAutoCorrected: Boolean, private val warn: S
 
     // ======== chapter 5 ========
     TOO_LONG_FUNCTION(false, "function is too long: split it or make more primitive"),
+    WRONG_FUNCTION_ARGUMENTS_ORDER(false, "lambda inside function parameters should be in the end"),
```

Comment:
```suggestion
    LAMDA_IS_NOT_LAST_PARAMETER(false, "lambda inside function parameters should be in the end"),
```
We may have other rules regarding parameters order in the future